### DISCUSSION TITLE
Update code-server version on railway

### DISCUSSION THREAD

**Discussion by TheScientist101** [coder/code-server#2902] [Q&A] [answered]
How can I update my code-server version in my container? I am hosting it on Railway, but it is on version 3.9.0, even though 3.9.1 is available.

**Comment by jsjoeio** [comment]
Great question! cc @bpmct

**Comment by bpmct** [comment, marked as answer]
Railway aggressively caches Docker images, so even though there is a new update on the :latest tag, you will want to change the first line of your repo's Dockerfile to

```
FROM codercom/code-server:3.9.1 
```

in order to use the latest version. I will be sure to document this and possibly talk to the folks at Railway about a solution so code-server can stay up to date :)

**Reply by bpmct** [reply]
in case you want to follow this :) https://github.com/cdr/deploy-code-server/issues/13
